conseil-api/src/main/scala/tech/cryptonomic/conseil/api/Conseil.scala

```diff
@@ -130,18 +135,18 @@ object Conseil
     * @param verbose flag to state if the server should log a more detailed configuration setup upon startup
     */
   def runServer(
-      metadataService: MetadataService,
-      conseilOperations: ConseilOperations,
-      server: ServerConfiguration,
-      platforms: PlatformsConfiguration,
-      metadataOverrides: MetadataConfiguration,
-      securityApi: SecurityApi,
-      verbose: VerboseOutput
+                 metadataService: MetadataService,
+                 conseilOperations: ConseilOperations,
+                 server: ConseilConfiguration,
+                 platforms: PlatformsConfiguration,
+                 metadataOverrides: MetadataConfiguration,
+                 securityApi: SecurityApi,
+                 verbose: VerboseOutput
   )(implicit executionContext: ExecutionContext, system: ActorSystem, mat: ActorMaterializer) = {
     val tezosDispatcher = system.dispatchers.lookup("akka.tezos-dispatcher")
 
     lazy val platformDiscovery = PlatformDiscovery(metadataService)
-    lazy val data = Data(metadataService, server, metadataOverrides, conseilOperations)(tezosDispatcher)
+    lazy val data = Data(metadataService, metadataOverrides, conseilOperations, server.maxQueryResultSize)(tezosDispatcher)
```

Comment:
I think that you can rename the dispatcher even now, if you prefer. It's not fundamentally tied to the platform.
It was named so mostly because it was tuned to tezos latencies/throughput.
But we can decide to rename later on when/if we need to introduce different dispatchers.

Reply:
Hm... do you mean, to make it more generic and also rename the section from the configuration?

Reply:
yes, if you think it makes sense, to start removing references to the "tezos" name when not strictly related.

Reply:
I just put it under `http` section, since it's related more to API/HTTP than tezos itself.